src/ngircd/messages.h

```diff
@@ -17,9 +17,9 @@
  * IRC numerics (Header)
  */
 
-#define RPL_WELCOME_MSG			"001 %s :Welcome to the Internet Relay Network %s"
-#define RPL_YOURHOST_MSG		"002 %s :Your host is %s, running version ngircd-%s (%s/%s/%s)"
```

Comment:
When changing directives in a format string, you have to update all its users, too, which still pass in tree arguments in this case!